styles/styles.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@600&family=Press+Start+2P&display=swap');
```

Comment:
you don't appear to be using the Press Start font?

Reply:
Resolved - misuse of google fonts